### PR TITLE
Update color values and internal color names to follow design mocks

### PR DESCRIPTION
**Pull request by gugalo**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-3709

**What was solved?**

Please refer to the git commits descriptions for details.

**P.S.1: I recommend to review commit-by-commit instead of all in one**
**P.S.2: Changes have been reviewed by Risto**

**Release notes:**

 - [x] Feature: **Slight UI color changes to the default color scheme. Changes should be barely visible**
 - [ ] Ignore
 - [ ] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

 - [ ] Is the feature sufficiently tested? All tests fixed? Necessary unit, acceptance, snapshots added? Check that at least new public classes & methods are covered with unit tests
 - [ ] Did you add logging beneficial for troubleshooting of customer issues?
 - [ ] **Did you add new logging?** We would like the logging between platforms to be similar. Refer to [**Logging from Android SDKs** → **Things to consider for newly added logs**](https://glia.atlassian.net/wiki/spaces/ENG/pages/3568861468/Logging+from+Android+SDKs#Things-to-consider-for-newly-added-logs) in Confluence for more information.

**Screenshots:**

### CODE REVIEW

File: widgetssdk/src/test/snapshots/images/com.glia.widgets.call_CallViewVideoSnapshotTest_onHold.png
Comment (DavDo) [inline widgetssdk/src/test/snapshots/images/com.glia.widgets.call_CallViewVideoSnapshotTest_onHold.png:1]: Is it expected to have a toolbar different from the background color?

Comment (gugalo) [reply widgetssdk/src/test/snapshots/images/com.glia.widgets.call_CallViewVideoSnapshotTest_onHold.png:1]: Good catch. I have missed that. 
Will look into it

File: widgetssdk/src/main/res/layout/entry_widget_powered_by_item.xml [hunk 1]
```diff
@@ -12,12 +13,13 @@
         android:layout_marginEnd="@dimen/glia_small"
         android:layout_marginBottom="@dimen/glia_medium"
         style="@style/Application.Glia.Caption"
-        android:textColor="?attr/gliaBaseShadeColor"
+        android:textColor="@color/glia_branding_color"
```
Comment (DavDo) [inline widgetssdk/src/main/res/layout/entry_widget_powered_by_item.xml:16]: will this be properly handled if the integrator has a `gliaBaseShadeColor` declaration in their theme?

File: widgetssdk/src/test/snapshots/images/com.glia.widgets.call_CallViewVideoSnapshotTest_onHold.png
Comment (gugalo) [reply widgetssdk/src/test/snapshots/images/com.glia.widgets.call_CallViewVideoSnapshotTest_onHold.png:1]: Fixed in f23a5e2

**Review by AndriiHorishniiMOC — APPROVED** [review]
LGTM!

**Pull request merged**